livesync/indico_livesync/handler.py

```diff
@@ -110,6 +107,10 @@ def _created(obj, **kwargs):
     _register_change(obj, ChangeType.created)
 
 
+def _event_deleted(obj, **kwargs):
```

Comment:
I'd rename the first argument to `event` and decorate it with `@unify_event_args`. Less code to change once we change the signal to pass the new Event object (which we could do already - the only case where the legacy object is needed is the signal handler that logs the ZODB OID, and that could simply be moved into the Conference.delete method).

Actually, you could even use `_deleted` for the event deleted signal and add the decorator there.